### DISCUSSION TITLE
Can this be adapted to Vite?

### DISCUSSION THREAD

**Discussion by pongells** [CycloneDX/cyclonedx-webpack-plugin#622] [Q&A] [answered]
I know nothing about webpack/vite plugins -- can this be adapted to work on Vite or are there plans for a Vite version?

Thanks

**Comment by jkowalleck** [comment, marked as answer]
Nope, this project itself will never support _vite_.
It's scoped for _webpack_.

If you want to implement an own tool, feel free to do so.
Consider announcing it in the [official tool center](https://cyclonedx.org/tool-center/).

If you want to publish your tool under the umbrella of [OWASP/CycloneDX](https://owasp.org/www-project-cyclonedx/), then come join the community:
- [Slack via invite](https://cyclonedx.slack.com/join/shared_invite/zt-cr5qsdhj-79iCCzgqgIKZXlKrqXVQrg#/shared-invite/email)
- [mailing group](https://groups.io/g/CycloneDX)
